### PR TITLE
Add snow depth tutorial

### PR DESCRIPTION
**Pull request by KarinaZikan**

### CODE REVIEW

**Comment by scottyhq** [comment]
Thanks @KarinaZikan . You'll want to add an entry in the book table of contents to your notebok:
https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/blob/098cc38323304dd9e50c389d560a1ae5a5e0fae7/book/_toc.yml#L31

`- file: tutorials/snow-depth/applications-tutorial-snow-depth`

**Comment by KarinaZikan** [comment]
The Codespell error is not actually an error

**Comment by scottyhq** [comment]
> The Codespell error is not actually an error 

Yes, sometimes it's great and sometimes it's annoying. spellchecking with code is hard. It's an optional check really, you can prevent checking certain files or suffixes by adding them here https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/blob/098cc38323304dd9e50c389d560a1ae5a5e0fae7/.github/workflows/qaqc.yaml#L35

**Comment by scottyhq** [comment]
Because of needing to access the files on JupyterHub I think we will actually just skip executing the notebook after all, and add an entry here:

https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/blob/098cc38323304dd9e50c389d560a1ae5a5e0fae7/book/_config.yml#L39-L40

**Comment by scottyhq** [comment]
Thanks @KarinaZikan this is looking very close to merge. Have a look at the preview to see that you're happy with how things are rendered https://deploy-preview-69--icesat-2-2023.netlify.app/tutorials/snow-depth/applications-tutorial-snow-depth .

File: book/tutorials/snow-depth/applications-tutorial-snow-depth.ipynb
Comment (scottyhq) [inline book/tutorials/snow-depth/applications-tutorial-snow-depth.ipynb:1]: For images to render correctly on the website, you need relative paths: (`Images/` -> `./Images`) 

(https://jupyterbook.org/en/stable/content/references.html#reference-content-files)

File: .github/workflows/qaqc.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ jobs:
         check_filenames: true
         check_hidden: true
         skip: '*.js,qaqc.yml'
+        skip: 'acp_lidar_box.geojson'
```
Comment (scottyhq) [inline .github/workflows/qaqc.yaml]:
```suggestion
        skip: '*.js,qaqc.yml,acp_lidar_box.geojson'
```

**Comment by scottyhq** [comment]
Actually... it seems JupyterBook does not do well with raw HTML for referencing images. Markdown allows you to do [my image](./pathto/image.png) if you don't mind scrapping the caption

Otherwise I think using this syntax will work (https://jupyterbook.org/en/stable/content/figures.html#figures)


````
```{figure} ../images/C-3PO_droid.png
---
height: 150px
name: directive-fig
---
Here is my figure caption!
```
````

**Review by scottyhq — APPROVED** [review]
Thanks @KarinaZikan and @zachghiaccio . Looks excellent. Be sure to check out this example https://icesat-2-2023.hackweek.io/tutorials/sliderule/parquet-s3.html  for sharing sliderule results among a bunch of people (everyone re-running the exact same request is fine for the tutorial but for projects it would be worth pointing out)

**Pull request merged**